File: Codekata-61.py
rows,num=input().split(" ")
rows=int(rows)
arr=[]
for i in range(rows):
	arr.extend([int(x) for x in input().split()])
flag=0
for j in arr:
	if j==int(num):
		print("yes")
		flag=1
		break
if(flag==0):
	print("no")

File: Codekata-61.js
const readline=require("readline");
const inp=readline.createInterface({
    input:process.stdin
});
const userInput=[]
inp.on("line",(data)=>{
    userInput.push(data);
})
inp.on("close",()=>{
    flag=0;
    for(var i=1;i<userInput.length;i++){
        for(var j=0;j<userInput[i].length;j++){
            if(userInput[i][j]==userInput[0][2]){
                console.log("yes");
                flag=1;
                break;
            }
        }
        if(flag===1){
            break;
        }
    }
    if(flag===0){
        console.log("no");
    }

});
